extensions/aria_extension_tosca/simple_v1_0/modeling/capabilities.py

```diff
@@ -162,6 +164,30 @@ def convert_capability_from_definition_to_assignment(context, presentation, cont
     return CapabilityAssignment(name=presentation._name, raw=raw, container=container)
 
 
+def merge_capability_definition(context, presentation, capability_definition,
+                                from_capability_definition):
```

Comment:
well, i might go with it regarding the first naming issue, but this is a new function, why not rename its arguments properly?